api/base/serializers.py

```diff
@@ -470,7 +474,18 @@ def format_filter(self, obj):
         qd = QueryDict(mutable=True)
         filter_fields = self.filter.keys()
         for field_name in filter_fields:
-            qd.update({'[{}]'.format(field_name): self.lookup_attribute(obj, self.filter[field_name])})
+            try:
+                # check if serializer method passed in
+                value = getattr(self.parent, self.filter[field_name])
+            except AttributeError:
+                value = self.lookup_attribute(obj, self.filter[field_name])
+            else:
+                value = value(obj)
```

Comment:
Before this line is evaluated, what is `value`? Is it the serializer method? If so, could you store that in a more specifically-named variable and then store the result into value? If not, I'm not sure what's going on here.